icu4j/pom.xml

```diff
@@ -517,6 +531,31 @@
          <module>tools/build</module>
       </modules>
     </profile>
+
+    <!-- Starting with JDK 9 the Java compiler supports the -release version.
```

Comment:
`-release` as you write here, or `--release` as you write in the PR description?

Reply:
Double, but `--` is not allowed in XML comments :-(

Reply:
I tried to find ways to escape, but I can't find anything.

I can do `\-\-`, and it will show that way. People will think that it's weird, but they will get it.

Or I can "cheat with Unicode" (put a ZWJ between the dashes, or use the minus sign (`U+2212`)).
But then one can't copy-paste it and have it work in command line.
True, it does not work now either...

Reply:
I went with U+2212